org.thecrossingchurch.CustomJobs/Jobs/HubspotIntegrationPatching.cs

```diff
@@ -0,0 +1,442 @@
+                // This implies that a merge occurred and we need to look at the alias table to figure out what the new Id is and update it
+                // After updating the ID, we need to find the person object and handle patching
+
+                // Setup for patching
+                // Look up hubspot defined type
+                DefinedTypeService definedTypeService = new DefinedTypeService( _context );
+                DefinedType hsDefinedType = definedTypeService.Get( 527 ); // as of 7/8/24 Dev is 528, Train is 527
+                AttributeValueService attributeValueService = new AttributeValueService( _context );
+                AttributeService attributeService = new AttributeService( _context );
+
+
+                //Schedule HubSpot update if 1:1 match
+                if ( person != null )
+                {
+                    current_id = person.Id;
+                    var url = $"https://api.hubapi.com/crm/v3/objects/contacts/{contacts[i].id}";
+                    Debug.WriteLine( "Contact Count: " + i + " of " + contacts.Count() );
+                    Debug.WriteLine( "URL: " + url );
+                    var properties = new List<HubspotPropertyUpdate>();
+
+
+                    foreach ( DefinedValue hsSyncDv in hsDefinedType.DefinedValues )
+                    {
+                        hsSyncDv.LoadAttributes();
+                        Dictionary<string, AttributeValueCache> dvAttributes = hsSyncDv.AttributeValues;
+                        
+                        string propertyOrAttribute = dvAttributes.GetValueOrNull("IsPropertyOrAttribute").Value;
+                        string hsKey = dvAttributes.GetValueOrNull("HubSpotAttributeKey").Value;
+                        string type = dvAttributes.GetValueOrNull( "Type" ).Value;
+                        string key = hsSyncDv.Value;
+                        var value = "";
+                        DateTime date;
+
+                        // Get person property or attribute
+                        if ( propertyOrAttribute == "Property" ) // is property
+                        {
+                            value = person.GetPropertyValue(key).ToStringSafe();
+                        }
+                        else // is attribute
+                        {
+                            var attributeQry = attributeService.Queryable().Where( a => a.EntityTypeId == 15 && a.Key == key ).AsNoTracking();
+                            try
+                            {
+                                Debug.WriteLine( "Type/Key: " + " " + type + " / " + hsKey );
+                                value = attributeValueService.Queryable().Where( av => av.EntityId == current_id ).Join( attributeQry, av => av.AttributeId, a => a.Id, ( av, a ) => av ).Select( av => av.Value ).AsNoTracking().FirstOrDefault().ToStringSafe();
+                            }
+                            catch
```

Comment:
This code shouldn't require using a try/catch. If there are exceptions, they need to be fixed, not ignored. Can you remove the try/catch and fix any potential errors (looks like this might just need some null handling).